src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -826,6 +850,11 @@ func (s *service) fetchTaggedIter(ctx context.Context, req *rpc.FetchTaggedReque
 		tagEncoder:      tagEncoder,
 		iOpts:           s.opts.InstrumentOptions(),
 		instrumentClose: instrumentClose,
+		totalDocsCount:  queryResult.Results.TotalDocsCount(),
+		nowFn:           s.nowFn,
+		fetchStart:      startTime,
```

Comment:
is this supposed to include the index lookup? Looks like it does (line 835)

Reply:
yes. it's total time of the fetch api call